src/esp/bindings_js/modules/vr_utils.js

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+/* global Module */
+
+export const VIEW_SENSORS = ["left_eye", "right_eye"];
+export const pointToArray = p => [p.x, p.y, p.z, p.w];
+
+export function getEyeSensorSpecs() {
+  const resWidth = 1024;
```

Comment:
I still don't think these should be hard-coded. Make them parameters so the caller code can choose.